backend/fms_core/filters.py

```diff
@@ -189,6 +190,11 @@ def latest_release_update_filter(self, queryset, name, value):
             latest_release_update=Max("readsets__release_status_timestamp")
         ).filter(latest_release_update__gte=value)
 
+    def latest_validation_status_update_filter(self, queryset, name, value):
+        return queryset.annotate(
+            latest_validation_update=Max("readsets__validation_status_timestamp")
+        ).filter(latest_validation_update__gte=value)
```

Comment:
We just changed the gte to gt for the release filter ... you should use the same here.

Reply:
Rebasing on master is the way to go